src/commands/cloud/deploy.ts

```diff
@@ -1,11 +1,8 @@
-import { Stream } from "stream";
-
 import { CommandModule } from "yargs";
 import tar from "tar";
 import FormData from "form-data";
-import axios from "axios";
 
-import { getDirs } from "../../utils";
+import { getDirs, makeCloudApiRequest} from "../../utils";
```

Comment:
```suggestion
import { getDirs, makeCloudApiRequest } from "../../utils";
```

Reply:
how didn't lint catch this?

Reply:
not sure, i think it cares about the ordering of the imports but not the spacing inside the brackets